Calmspace/DisplayViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  DisplayViewController.swift
+//  Calmspace
+//
+//  Created by Admin on 17/10/2023.
+//
+
+import UIKit
+import WebKit
+
+class DisplayViewController: UIViewController, WKNavigationDelegate {
+    
+    var url: URL?
+    
+    
+    @IBOutlet weak var webView: WKWebView!
```

Comment:
Naming sometimes is referred to as the hardest problem in programming 🙂 
A rule of thumb is never name a variable by its type name, e.g.:

```swift
let string = "Hello!" // bad
let greeting = "Hello!" // good
let buttonTitle = "Press me" // good

let array = [0, 1, 3] // bad
let indiciesArray = [0, 1, 3] // better, but still `array` is unnecessary
let selectedIndices = [0, 1, 3] //good
```

A name of a variable has to be useful: informative, and help anyone reading this code to instantly understand what is stored inside. I saw that the sample code from the courses that you saw did not always have good variable naming. We can do better. Try to come up with good names for all the properties in this PR.
Try to find good names to

Reply:
On another note, `webView` might be an exception, it is common to call a variable that holds `WKWebView` like this. But other properties need to get proper names.